tests/Doctrine/Tests/DBAL/Schema/ColumnTest.php

```diff
@@ -56,6 +57,16 @@ public function testToArray()
     }
 
     /**
+     * @group legacy
+     * @expectedDeprecation The "unknown_option" option is not supported, setting it is deprecated and will cause an exception in 3.0
+     * @doesNotPerformAssertions
```

Comment:
The build passed from red to green after adding this line... not sure why: https://github.com/symfony/symfony/pull/21896